packages/relay/src/lib/services/ethService/contractService/ContractService.ts

```diff
@@ -255,18 +245,6 @@ export class ContractService implements IContractService {
           return CommonService.redirectBytecodeAddressReplace(address);
         } else if (result.type === constants.TYPE_CONTRACT) {
           if (result.entity.runtime_bytecode !== constants.EMPTY_HEX) {
-            const prohibitedOpcodes = ['CALLCODE', 'DELEGATECALL', 'SELFDESTRUCT', 'SUICIDE'];
-            const opcodes = disassemble(result.entity.runtime_bytecode);
```

Comment:
The `disassemble` constant and the `@ethersproject/asm` package could be removed altogether since this has been removed